src/pages/BatchSearchResults.vue

```diff
@@ -24,7 +24,20 @@
                 {{ $t('batchSearch.projects') }}
               </dt>
               <dd>
-                {{ batchSearch.projects.map((project) => project.name).join(', ') }}
+                <router-link
+                  v-for="(project, index) in batchSearch.projects"
+                  :key="project.name"
+                  :to="{
+                    name: 'search',
+                    query: {
+                      q: '*',
+                      indices: project.name
+                    }
+                  }"
+                  class="batch-search-results__info__project-link"
+                >
+                  {{ project.name }}<span v-if="isNotLastArrayItem(index, batchSearch.projects.length)">,</span>
```

Comment:
I think it would be more pertinent  if the comma is outside of the link unless it's looking weird

Reply:
Good idea, I changed it